### PR TITLE
Add `buildFromScratch`

### LINKED ISSUE #52: Alternative `build` for building a record from scratch
Hi everyone! 👋

Would anyone be opposed to a `build' :: Builder {} { | row } -> { | row }` function to build a record from scratch? The implementation is only `flip build $ {}` but it’s a convenient little function to compose in some pointfree scenarios.

### PR DESCRIPTION
**Pull request by kl0tl**
Close https://github.com/purescript/purescript-record/issues/52.

### CODE REVIEW

File: src/Record/Builder.purs [hunk 1]
```diff
@@ -45,6 +46,10 @@ newtype Builder a b = Builder (a -> b)
 build :: forall r1 r2. Builder (Record r1) (Record r2) -> Record r1 -> Record r2
 build (Builder b) r1 = b (copyRecord r1)
 
+-- | Build a record from scratch.
+buildFromScratch :: forall r. Builder (Record ()) (Record r) -> (Record r)
+buildFromScratch = flip build $ {}
```
Comment (hdgarrood) [inline src/Record/Builder.purs]: I think the `$` is unnecessary here.

Comment (kl0tl) [reply src/Record/Builder.purs]: It was indeed! I removed it.

File: src/Record/Builder.purs [hunk 2]
```diff
@@ -45,6 +46,10 @@ newtype Builder a b = Builder (a -> b)
 build :: forall r1 r2. Builder (Record r1) (Record r2) -> Record r1 -> Record r2
 build (Builder b) r1 = b (copyRecord r1)
 
+-- | Build a record from scratch.
+buildFromScratch :: forall r. Builder (Record ()) (Record r) -> (Record r)
```
Comment (hdgarrood) [inline src/Record/Builder.purs]: Oh oops sorry, another nit: the parens around the Record r at the end shouldn’t be necessary.

**Comment by hdgarrood** [comment]
Looks good to me, does anyone else have any thoughts?

**Comment by justinwoo** [comment]
Only question is why "scratch" and not "empty"? We have a Record.empty right?

File: src/Record/Builder.purs (hunk 2 above)
Comment (kl0tl) [reply src/Record/Builder.purs]: Those parens are no more.

**Comment by kl0tl** [comment]
I’m happy to rename to whatever is preferred. Please note that there’s currently no `Record.empty` though.

**Comment by kl0tl** [comment]
Should I rename to `buildFromEmpty` or close that PR? It could also be the occasion to add `Record.empty`.

**Comment by hdgarrood** [comment]
I don’t think we need a Record.empty when we already have `{}`.

**Review by hdgarrood — APPROVED** [review]

**Pull request merged**